variables.tf

```
@@ -17,18 +17,25 @@ variable "alarm_actions" {
 variable "warning_database_usage_threshold" {
   description = "The threshold for warning level database usage alarm"
   type        = number
-  default     = 90
+  default     = null
 }
 
 variable "critical_database_usage_threshold" {
   description = "The threshold for critical level database usage alarm"
   type        = number
-  default     = 96
+  default     = null
 }
 
 variable "sns_topic_arn" {
-  type        = string
   description = "The SNS topic ARN to notify."
+  type        = string
+  default     = null
+}
+
+variable "alerts_enabled" {
+  description = "A variable to control the creation of resources. If set to false, no resources will be created."
```

```suggestion
  description = "Create CloudWatch alerts for provisioned clusters. Please set `sns_topic_arn`"
```

Please do not write obvious description, it should add value to the name of variable.

---

or 

`description = "Determines if alert-related resources should be created. When false, associated alerting resources are skipped. Please set `sns_topic_arn`"`